includes/wizards/class-setup-wizard.php

```diff
@@ -296,34 +380,70 @@ public function api_update_theme_with_mods( $request ) {
 		$theme = $request['theme'];
 		Starter_Content::set_theme( $theme );
 
+		// Set homepage pattern.
+		if ( isset( $request['theme_mods']['homepage_pattern_index'] ) ) {
+			$homepage_pattern_index = (int) $request['theme_mods']['homepage_pattern_index'];
+			$homepage_pattern       = $this->get_homepage_patterns( $homepage_pattern_index );
+			if ( false !== $homepage_pattern ) {
+				$homepage_id = get_option( 'page_on_front', false );
+				if ( $homepage_id && false !== $homepage_pattern ) {
```

Comment:
Nit: the `false !== $homepage_pattern` condition seems redundant here since it's already wrapped in the same condition on line 387 above.

Reply:
Good suggestion, added in b4ff0ca

Reply:
True, removed in b57691d178539869494cd0a8824ba250b1dc7d95